orc8r/cloud/go/obsidian/swagger/spec/combine.go

```diff
@@ -146,7 +146,7 @@ func makeAllYAMLReferencesLocal(yamlContents []string) []string {
 	// Match on any yml reference to file_name_here.foo.bar#/baz
 	// and change those references to #/baz (strip the prefix)
 	// e.g. $ref: 'foo_bar_baz.blah#/asdf' -> $ref: '#/asdf'
-	ymlRefRe := regexp.MustCompile(`(\$ref:\s*)['"].+(#/.+)['"]`)
+	ymlRefRe := regexp.MustCompile(`(\$ref:\s+)['"]?.*(#/[^"'\s]+)["']?`)
```

Comment:
Nit: put all single vs. double quotes in the same order. So change the first `['"]` to `["']`